Refactor "toot" and "post" to reflect nouns and verbs

vollkorntomate opened the pull request:
### Summary

I refactored the usage of the string "toot" and "post" and split them into separate localization keys `"toot.action"` and `"toot.noun"` (`"post.action"`/`"post.noun"`), since it wasn't flexible enough to handle both the action of tooting/posting as well as the noun/object of a toot/post. This caused many translations to use the same word for both things, resulting in wrong grammar.

Since (of the existing translations) I only know German, English, French and Spanish, the other translations might be inaccurate and need fixing.

- [x] I have signed [Metabolist's Contributor License Agreement](https://metabolist.org/cla)

vollkorntomate committed:
Refactor "toot" and "post" to reflect nouns and verbs

In English, "toot" and "post" describe both the noun/object and the action. In other languages, there might be a difference.

jzzocc commented:
Would you mind changing this to limit the scope to just changing the keys and submit updated strings at https://crowdin.com/project/metatext instead? I will get around to adding InfoPlist.strings there soon as well, thanks for bringing that to my attention

vollkorntomate commented:
I updated the PR to only change the "toot" and "post" word usage, and fixed the translations for German in the Crowdin project, just like you suggested.
